dev/deps/spark-deps-hadoop-2.2

```diff
@@ -115,7 +115,7 @@ libfb303-0.9.2.jar
 libthrift-0.9.2.jar
 log4j-1.2.17.jar
 lz4-1.3.0.jar
-mesos-0.21.1-shaded-protobuf.jar
+mesos-0.22.2-shaded-protobuf.jar
```

Comment:
We have another PR in flight that will update this to 0.28. Is this going to be OK? maybe will merge that first